config/projects.yaml

```diff
@@ -145,16 +145,18 @@ classification-bundle:
     branches:
         master:
             php: ['7.3', '7.4']
+            php_extensions: [mongodb-1.9.0]
             variants:
                 symfony/symfony: ['4.4']
                 sonata-project/admin-bundle: ['3']
-            services: [mongodb]
+            tools: [mongodb, pecl]
```

Comment:
```suggestion
            tools: [pecl]
```
same for others.

Reply:
Tools are for ubuntu services:
https://github.com/sonata-project/dev-kit/blob/1ab98d2bd78b868d8d9fe554a286da90cd56ac5d/templates/project/.github/workflows/test.yaml.twig#L26-L33

`php_extensions` is for php.

Reply:
I will check it after 15min break.

Reply:
@VincentLanglet
Done.